backend/LexBoxApi/Services/FwHeadlessClient.cs

```diff
@@ -50,4 +50,17 @@ public async Task<bool> CrdtSync(Guid projectId)
             await response.Content.ReadAsStringAsync());
         return null;
     }
+
+    public async Task<bool> DeleteRepo(Guid projectId, CancellationToken cancellationToken = default)
+    {
+        var response = await httpClient.DeleteAsync($"/api/manage/repo/{projectId}", cancellationToken);
+        if (response.IsSuccessStatusCode)
```

Comment:
Assuming we start returning 404's from the delete endpoint
```suggestion
        if (response.IsSuccessStatusCode || 404)
```